### PR TITLE
Cleaned up unused,obsolete and unsupported platforms and toolchains

### PR DESCRIPTION
**Pull request by cw2**
Removed WINCE, BLACKFIN, Renesas, ARC, old ARM toolchain, GCCOP...

Signed-off-by: Stanislav Šimíček <cw2@users.noreply.github.com>

### CODE REVIEW

File: targets/os/win32/nanoCLR/CLRStartup.cpp [hunk 1]
```diff
@@ -167,7 +167,7 @@ struct Settings
```
Comment (josesimoes) [inline targets/os/win32/nanoCLR/CLRStartup.cpp]: Please add empty line at lhe end of the file to keep GitHub diff tool happy.

File: src/CLR/Include/TinySupport.h [hunk 2]
```diff
@@ -16,7 +16,7 @@ UINT32 SUPPORT_ComputeCRC(const void* rgBlock, int nLength, UINT32 crc);
```
Comment (josesimoes) [inline src/CLR/Include/TinySupport.h]: Please add empty line at lhe end of the file to keep GitHub diff tool happy.

File: src/CLR/Include/TinyHAL_Types.h [hunk 3]
```diff
@@ -10,16 +10,12 @@
```
Comment (josesimoes) [inline src/CLR/Include/TinyHAL_Types.h]: Please add empty line at lhe end of the file to keep GitHub diff tool happy.

File: src/CLR/Include/TinyHAL.h [hunk 4]
```diff
@@ -579,7 +579,7 @@ extern void HAL_AssertEx();
         #define _SIDE_ASSERTE(i) { if(!(i)) HAL_AssertEx(); }
```
Comment (josesimoes) [inline src/CLR/Include/TinyHAL.h]: There is an unusual number of empty lines in this file, if there is no purpose for those please clean up the file.
Also add an empty line at lhe end of the file to keep GitHub diff tool happy.

File: src/CLR/Include/TinyCLR_Types.h [hunk 5]
```diff
@@ -84,7 +84,7 @@ enum CLR_OPCODE
 CEE_COUNT,        /* number of instructions and macros pre-defined */
```
Comment (josesimoes) [inline src/CLR/Include/TinyCLR_Types.h]: Please add empty line at lhe end of the file to keep GitHub diff tool happy.

File: src/CLR/Include/TinyCLR_Runtime.h [hunk 6]
```diff
@@ -21,10 +21,6 @@ struct CLR_RADIAN
```
Comment (josesimoes) [inline src/CLR/Include/TinyCLR_Runtime.h]: Please add empty line at lhe end of the file to keep GitHub diff tool happy.

File: src/CLR/Debugger/Debugger.cpp [hunk 7]
```diff
@@ -956,14 +956,7 @@ bool CLR_DBG_Debugger::Debugging_Execution_ChangeConditions( WP_Message* msg, vo
```
Comment (josesimoes) [inline src/CLR/Debugger/Debugger.cpp]: There is an unusual number of empty lines in this file, if there is no purpose for those please clean up the file.
Also add an empty line at lhe end of the file to keep GitHub diff tool happy.

File: src/CLR/Include/TinyHAL.h (hunk 4 above)
Comment (cw2) [inline src/CLR/Include/TinyHAL.h]: This is on purpose, for easy line-by-line (unaligned) compare with the original. It will be cleaned up...

**Comment by cw2** [comment]
Please don't merge yet, I'll fix the line ending...

**Comment by josesimoes** [comment]
> Please don't merge yet, I'll fix the line ending...

Sure thing! I just gave my OK ahead so you can merge this right after you submit the fix. 😄

**Comment by cw2** [comment]
I'll fix line endings in a separate PR, for all source files.

**Pull request merged**